dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -1096,7 +1096,47 @@ public void testGenerateDOIFrozenVersion() throws ApiException {
             assertTrue(ex.getResponseBody().contains(NO_ZENDO_USER_TOKEN));
 
         }
+    }
+
+    /**
+     * This tests that a published workflow, when launched, attempts to validate descriptor checksums.
+     * That cases that are tested: Checksum matches, checksum doesn't match, checksum doesn't exist
+     *
+     */
+    @Test
+    public void launchWorkflowChecksumValidation() {
+
+        // register and publish a workflow
+        Client.main(
+            new String[] { "--config", ResourceHelpers.resourceFilePath("config_file2.txt"), "workflow", "manual_publish", "--repository",
+                "md5sum-checker", "--organization", "DockstoreTestUser2", "--git-version-control", "github",
+                "--workflow-path", "/checker-workflow-wrapping-tool.cwl", "--descriptor-type", "cwl", "--workflow-name", "checksumTester", "--script" });
+
+        // ensure checksum validation is acknowledged, and no null checksums were discovered
+        systemOutRule.clearLog();
+        Client.main(new String[] {"--config", ResourceHelpers.resourceFilePath("config_file2.txt"), "workflow", "launch", "--entry",
+            "github.com/DockstoreTestUser2/md5sum-checker/checksumTester", "--json", ResourceHelpers.resourceFilePath("md5sum_cwl.json"), "--script" });
+        assertTrue("Output should indicate that checksums have been validated",
+            systemOutRule.getLog().contains("Validated checksums") && !systemOutRule.getLog().contains("Cannot validate the checksum of the locally downloaded descriptor."));
```

Comment:
Ditto for these tests

Reply:
[Fixed](https://github.com/dockstore/dockstore-cli/blob/7f6c85a4a186819af1abeaf2f795939c1fb693b4/dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java#L1110). Added final strings at start of tests.